{
  "apiGeneralError": {
    "user-already-exists": "Пользователь существует",
    "Internal server error": "Internal server error",
    "wrong-credentials": "Неправильные учетные данные",
    "account-not-confirmed": "Почта не подтверждена",
    "unauthorized": "Неавторизовация",
    "session-has-expired": "Срок действия сеанса истек"
  },
  "generalError": {},
  "unknownError": {
    "unknownError": "Неизвестная ошибка"
  }
}
